### DISCUSSION TITLE
Application not found after upgraded the version to 4.8.0

### DISCUSSION THREAD

**Discussion by bmm-alc** [wazuh/wazuh#24087] [General]
Hello,

I'm operating a wazuh cluster on kubernetes, and I upgraded to 4.8.0. all the components are up but after being successfully logged I get this error message and I cannot access the dashboard

![image](https://github.com/wazuh/wazuh/assets/80455946/fa6821ec-faf5-4856-ba11-9ccfe85c6c2e)

I did not see any particular steps to do prior or upon 4.8.0 installed in the documentation

**Comment by AlexH-HankIT** [comment]
I'm having the same issue. I run a single node deployment using Docker.

Edit: 
Fix is here: https://old.reddit.com/r/Wazuh/comments/1df1erl/wazuh_upgrade_to_v48_issues/ or https://github.com/wazuh/wazuh/issues/24090
